app/src/main/java/app/grapheneos/camera/ui/SettingsDialog.kt

```diff
@@ -54,8 +55,13 @@ class SettingsDialog(val mActivity: MainActivity) :
     private var aRToggle: ToggleButton
     var torchToggle: ToggleButton
     private var gridToggle: ImageView
+
     var videoQualitySpinner: Spinner
+    var videoFrameRateSpinner: Spinner
+
     private lateinit var vQAdapter: ArrayAdapter<String>
+    private lateinit var vFRAdapter: ArrayAdapter<String>
```

Comment:
This variable name is too short

Reply:
I have renamed "vFRAdapter" to "videoFrameRateAdapter" in the recently pushed changes